src/renderer/i18n/en.ts

```
@@ -31,6 +31,9 @@
       error: "Failed to save settings",
     },
   },
+  beat: {
+    badge: beat_badge,
```

ESLint

[error] 'beat_badge' is not defined.

(no-undef)